imports/plugins/included/product-variant/containers/gridItemNoticeContainer.js

```diff
@@ -23,9 +23,7 @@ const wrapComponent = (Comp) => (
 
       for (const topVariant of topVariants) {
         const inventoryThreshold = topVariant.lowInventoryWarningThreshold;
-        const inventoryQuantity = ReactionProduct.getVariantQuantity(topVariant);
-
-        if (inventoryQuantity !== 0 && inventoryThreshold >= inventoryQuantity) {
+        if (topVariant.inventoryAvailableToSell !== 0 && inventoryThreshold >= topVariant.inventoryAvailableToSell) {
```

Comment:
It should be `inventoryAvailableToSell > 0` in case it's negative.

Also it should be `inventoryThreshold > inventoryAvailableToSell` to match server, assuming server calculation is correct.